### DISCUSSION TITLE
About LES filter

### DISCUSSION THREAD

**Discussion by bronise777** [AMReX-Combustion/PeleC#843] [General]
I have a question. Why is only the hydro flux filtered when explicit filtering is used, but not the diffusion flux? And, are there any examples of filter here? Thanks for your answer.

**Comment by marchdf** [comment]
For a good answer I would refer you to a seminal work for explicit filtering: The use of explicit filters in large eddy simulation, T.S. Lund, Computers & Mathematics with Applications, http://linkinghub.elsevier.com/retrieve/pii/S0898122103900198, particularly bottom half of p 607.

**Reply by bronise777** [reply]
Thanks for your recommendation, I have read this paper carefully.
Mesh size and finite difference are considered implicit filters by default when calculating, and it is the first filter. The explicit filter is to deal with the problem of high frequency product of nonlinear terms and is the second filter.
Whether my understanding is correct?

**Reply by bronise777** [reply]
What's different about the new subgrid model? Can we also use something like Smagorinsky's model to solve it?

**Reply by marchdf** [reply]
> Whether my understanding is correct?

yes that's right.

Not sure I understand your other questions. Smagorinsky is a subgrid scale model. So is the newer WALE and Vreman ones. I would recommend a literature review one those to understand the differences. It's a quick search away.

**Reply by bronise777** [reply]
I also have a question, in the program the whole hydro flux is explicitly filtered, but here the formula does not seem to filter the pressure term. Does the hydro flux in pelec include the pressure term?
![微信截图_20240918093510](https://github.com/user-attachments/assets/6222e335-63ab-41b1-8e4b-e8f1ff6b3343)

**Reply by bronise777** [reply]
![微信截图_20240918162223](https://github.com/user-attachments/assets/a1ad4f04-8d90-4aac-bcdf-039295f0f380)
And, is this new les term obtained by explicitly filtering the original les term?

**Reply by marchdf** [reply]
Pressure is included in the hydro flux. If you believe this is a mistake and it makes a difference, feel free to change the code to suit your needs.

I am not sure what you mean by "les term".

**Reply by bronise777** [reply]
I see in LES.cpp that this SGS term is also filtered if explicit filtering is used. Is this the **_T’_**  from the above formula? The filtered SGS item is **_T'_**, and the original sgs item is _**T**_. Whether my understanding is correct?
  if (use_explicit_filter) {
    les_filter.apply_filter(LESTerm, filtered_les_source);
    LESTerm.define(
      grids, dmap, NVAR, filtered_les_source.nGrow(), amrex::MFInfo(),
      Factory());
    amrex::MultiFab::Copy(
      LESTerm, filtered_les_source, 0, 0, NVAR, filtered_les_source.nGrow());
  }

**Reply by marchdf** [reply]
yes it is. Thank you for clarifying.

**Reply by bronise777** [reply]
Great! Thanks again!
